MosipNexus/ingestion/store.py

```
@@ -423,14 +468,27 @@ def prepare_generic_documents(file_path, source_type_label: str) -> list[Documen
+            ESIGNET_COLLECTION,
+            embeddings,
+        )
 
     # ── Seed crawl state so run_update.py knows what's already ingested ────────
     print("\nSeeding crawl state for future incremental updates...")
     url_hashes = {
-        doc["url"]: content_hash(doc.get("content", ""))
-        for doc in raw_docs
-        if doc.get("content", "").strip()
-    }
+    doc.metadata["source"]: _hash_content(doc.page_content)
```

Ruff (0.15.20)

[error] Undefined name `_hash_content`

(F821)